_data/doomSDey/doomSDey.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sudipta Dey # No longer than 28 characters
+institution: Sikkim Manipal Institute of Technology # no longer than 58 characters
+profile_pic: doomSDey.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```

Comment:
Hey @doomSDey,
Please change the following in your PR,
1. Looks like your image is of 1602x2314 px. Make your image a 1:1 size. (i.e, either 1602 x 1602 px or any other size)
2. Correct the image file name with the proper extension. In your case, it is to be `doomSDey.JPG` and not `doomSDey.jpg`